ang/civicase/CaseDetailsPeopleTab.js

```diff
@@ -79,11 +79,24 @@
       if (onlyChecked || $scope[tab + 'SelectionMode'] === 'checked') {
         return _.collect(_.filter($scope[tab], {checked: true}), idField);
       } else if ($scope[tab + 'SelectionMode'] === 'all') {
-        return _.collect($scope[tab], idField);
+        return _.collect(_.filter($scope[tab], function (el) {
+          return el.contact_id;
+        }), idField);
       }
       return [];
     };
 
+    /**
+     * Filter the records on basis of contact_role key
```

Comment:
There is no trace of `contact_role` in the function implementation

Reply:
Ooops! It should be `contact_id`. Changed. Please have a look.